components/modal/modal.js

```diff
@@ -184,6 +186,7 @@ Modal.defaultProps = {
   closeOnOverlayClick: true,
   footer: null,
   fullscreen: false,
+  mods: [],
```

Comment:
possible reference issue, please put an array in the render function:
`const { active, fullscreen, children, mods = [] } = this.props;`